DxDiagram improve Expr descriptions (#620)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/edges/fromExpr.md

The current edge's data object.

---

> Specify this option if your source data is contained in two ([node and edge](/Documentation/Guide/Widgets/Diagram/Data_Binding/#Node_and_Edge_Arrays)) data sources.

Reviewer: ```suggestion
Specify this option if you use [node and edge](/Documentation/Guide/Widgets/Diagram/Data_Binding/#Node_and_Edge_Arrays)) data sources.
```
